openfl-tutorials/interactive_api/PyTorch_Lightning_MNIST_GAN/envoy/sd_requirements.txt

```diff
@@ -0,0 +1,8 @@
+numpy
+pillow
+pytorch-lightning>=1.3
+pynvml==11.4.1
+tensorflow==2.7.0
```

Comment:
Do we really need TensorFlow/PyTorch Lightning/Torchmetrics in ShardDescriptor requirements?